packages/blade/.gitignore

```
@@ -54,6 +54,8 @@ yarn-error.log
 
 # Bundle artifact
 *.jsbundle
+.size-limit.json
+PRBundleSizeStats.json
```

Lets use camelCase instead of PascalCase. Its more consistent with our practices
```suggestion
prBundleSizeStats.json
```